pkg/pagerduty/README.md

```diff
@@ -23,17 +23,29 @@ func GetPDClient() (pagerduty.Client, error) {
 ```
 
 ## make `cadctl` load PD creds
-### if you can `pagerduty-cli` installed
+### if you have `pagerduty-cli` installed
 you can run the oneliner:
 ```
-export CAD_PD=$(jq  .subdomains[].legacyToken ~/.config/pagerduty-cli/config.json -r) 
+export CAD_PD_TOKEN=$(jq  .subdomains[].legacyToken ~/.config/pagerduty-cli/config.json -r) 
 ```
 and the token will be used
 
 ### if you wish to load a token from the UI
-use the link to get an token, and bind it to the envvar https://support.pagerduty.com/docs/api-access-keys#generate-a-user-token-rest-api-key
+use the link to get a token, and bind it to the envvar https://support.pagerduty.com/docs/api-access-keys#generate-a-user-token-rest-api-key
 
-## Manutally test on an incident
+### Create the secret 
+
+* CAD_ESCALATION_POLICY: refers to the escalation policy CAD should use to escalate the incident to
+* CAD_PD_EMAIL: refers  to the email for a login via mail/pw credentials
+* CAD_PD_PW: refers to the password for a login via mail/pw credentials
+* CAD_PD_TOKEN: refers to the generated private access token for token-based authentication
+* CAD_PD_USERNAME: refers to the username in case username/pw credentials should be used
+* CAD_SILENT_POLICY: refers to the silent policy CAD should use if the incident shall be silent
+* PD_SIGNATURE: refers to the PagerDuty webhook signature (HMAC+SHA256)
+* X_SECRET_TOKEN: refers to our custom Secret Token for authenticating against our pipeline
```

Comment:
this should be in the secret file, not in the readme (as close as we can to the place they need to be changed

Reply:
Oh the secret is already up to date: https://github.com/openshift/configuration-anomaly-detection/blob/main/deploy/task-cad-checks-secrets-pd.yaml

Or shall I add comments behind each line?

Reply:
Yes, that would make the changing of variables much cleaner

Reply:
fixed.. I added the comments to the secret